fala/apps/adviser/forms.py

```diff
@@ -27,14 +27,19 @@ class AdviserSearchForm(forms.Form):
     page = forms.IntegerField(required=False, widget=forms.HiddenInput())
 
     postcode = forms.CharField(
-        label=_("Enter postcode"),
+        label=_("<label class='govuk-label govuk-label--s'>Postcode</label>"),
         max_length=30,
-        help_text=_("For example, <span class='notranslate' translate='no'>SW1H 9AJ</span>"),
+        help_text=_("For example, <span class='notranslate' translate='no'>SW1H</span>"),
```

Comment:
I know this a copy. They're being a little naughty here. While they're using inline HTML here, which isn't great, the way they set up the marcos in form.html makes working with this a bit of a pain, so another solution here is to use `django.utils.safestring`

```
from django.utils.safestring import mark_safe
help_text = mark_safe("<label class='govuk-label govuk-label--s'>Postcode</label>")
```